docs/commands.md

```diff
@@ -0,0 +1,58 @@
+# :: Commands
+
+The [core toolkit package](https://github.com/actions/toolkit/tree/master/packages/core) offers a number of convenience functions for
+setting results, logging, registering secrets and exporting variables across actions. Sometimes, however, its useful to be able to do
+these things in a script or other tool.
+
+To allow this, we permit a special `::` syntax which, if logged to stdout, will allow the runner to interpret your commands.
+The following commands are all supported:
+
+### Set an environment variable
+
+To set an environment variable, use `::set-env`:
+
+`echo ::set-env name=FOO::BAR`
```

Comment:
For easier scanning and nicer formatting, what do you think about making these block-level code blocks?

`echo ::set-env name=FOO::BAR`

vs:

```
echo ::set-env name=FOO::BAR
```